Example/RxRealm/ViewController.swift

```diff
@@ -32,23 +35,28 @@ class ViewController: UIViewController {
         //Observable<Array<Lap>>
         let laps = realm.objects(Lap).sorted("time", ascending: false).asObservableArray()
         
-        laps.map {array in
-            return array.prefix(3) //get array slice of the 3 most recent items
-        }
+        laps
         .bindTo(tableView.rx_itemsWithCellIdentifier("Cell", cellType: UITableViewCell.self)) {row, element, cell in
             cell.textLabel!.text = formatter.stringFromDate(NSDate(timeIntervalSinceReferenceDate: element.time))
         }.addDisposableTo(bag)
 
-        //start adding laps
-        addLap()
-    }
-    
-    func addLap() {
-        let realm = try! Realm()
-        try! realm.write {
-            realm.add(Lap())
-        }
+        
+        addOneItemButton.rx_tap
+            .map {
+                return Lap()
+            }
+            .bindTo(realm.rx_addObject())
```

Comment:
let me play a bit with this today - looking at RxCocoa most of the bindable sinks are `var`s but on the other hand they aren't handling different types like we need for RxRealm either ...